installer/tue-install.bash

```diff
@@ -42,30 +39,28 @@ else
     fi
 fi
 
-if [ ! -d $TUE_ENV_TARGETS_DIR ]
+if [ ! -d "$TUE_ENV_TARGETS_DIR" ]
 then
     echo "[tue-get] 'TUE_ENV_TARGETS_DIR' $TUE_ENV_TARGETS_DIR doesn't exist"
+    # shellcheck disable=SC1078,SC1079
     echo """To setup the default tue-env targets repository do,
 
 tue-env init-targets https://github.com/tue-robotics/tue-env-targets.git
 """
     exit 1
 else
-    current_url=$(git -C $TUE_ENV_TARGETS_DIR config --get remote.origin.url)
-    new_url=$(_github_https_or_ssh $current_url)
+    current_url=$(git -C "$TUE_ENV_TARGETS_DIR" config --get remote.origin.url)
+    new_url=$(_github_https_or_ssh "$current_url")
 
     if [ "$current_url" != "$new_url" ]
     then
-        git -C $TUE_ENV_TARGETS_DIR remote set-url origin $new_url
+        git -C "$TUE_ENV_TARGETS_DIR" remote set-url origin "$new_url"
         echo -e "[tue-env-targets] Origin has switched to $new_url"
     fi
 
     echo -en "[tue-env-targets] Updating targets... "
-    git -C $TUE_ENV_TARGETS_DIR pull --ff-only --prune
-
-    error_code=$?
 
-    if [ ! $error_code -eq 0 ] && [ -z "$CI" ]
+    if ! git -C "$TUE_ENV_TARGETS_DIR" pull --ff-only --prune
```

Comment:
May get stuck in CI

Reply:
Why?

Reply:
If the `if` block is executed in CI then it will prompt for input

Reply:
You have removed the CI check from the if condition

Reply:
fixed